examples/tutorials/starting/analysis_2.py

```diff
@@ -394,8 +394,11 @@
 #
 
 energy_bounds = [1, 10] * u.TeV
-spec.plot(energy_bounds=energy_bounds, energy_power=2)
-ax = spec.plot_error(energy_bounds=energy_bounds, energy_power=2)
+sed_type = "e2dnde"
```

Comment:
I would have rather left the explicit chain in the function call. If you copy only one cell then this will break.

Reply:
Agree, but then shouldn't we also do it for `energy_bounds`?